### DISCUSSION TITLE
Not able to view my profile on linktree

### DISCUSSION THREAD

**Discussion by Pritam0077** [EddieHubCommunity/LinkFree#1214] [Q&A] [answered]
I have made my profile and submitted the pr but I am not able to view my profile on Linktree. Can someone help me out?

**Comment by suitupstan** [comment, marked as answer]
After your PR is merged, you need to wait till the next production deploy initiated by a maintainer 🙂

**Reply by Pritam0077** [reply]
Ok thanks

**Reply by KapilKumar7** [reply]
> After your PR is merged, you need to wait till the next production deploy initiated by a maintainer 🙂

And How Many days it generally takes?

**Reply by SaptarshiSarkar12** [reply]
> > After your PR is merged, you need to wait till the next production deploy initiated by a maintainer slightly_smiling_face
> 
> And How Many days it generally takes?

It depends on when it will be deployed. Stay tuned with this project!
